apps/server/src/node/schema/node.shema.ts

```diff
@@ -0,0 +1,29 @@
+import { Prop, Schema, SchemaFactory } from "@nestjs/mongoose";
+import mongoose, { SchemaTypes, Types } from "mongoose";
+
+export interface INode {
+    id: string;
+    title?: string;
+    content?: string;
+}
+
+interface ISchemaNode extends INode {
+    workspaceId: Types.ObjectId
+}
+
+@Schema({ timestamps: true })
+export default class Node implements ISchemaNode {
+  @Prop({ required: true })
+  id: string;
+
+  @Prop({ maxlength: 80 })
+  title?: string;
+
+  @Prop({ maxlength: 300 })
+  content?: string;
+
+  @Prop({ required: true, type: SchemaTypes.ObjectId })
```

Comment:
We do not need workspaceId at every node level. We need this info at top level of mindmap.